wolfCLU/README.md

```diff
@@ -7,7 +7,7 @@ This is the wolfSSL: Command Line Utility (wolfCLU).
 To use this feature, please configure and install wolfssl with the following commands:
 
 ```
-./configure --enable-pwdbased --enable-opensslextra --enable-keygen --enable-ed25519 --enable-certgen
+./configure --enable-pwdbased --enable-opensslall --enable-keygen --enable-ed25519 --enable-certgen --enable-certreq --enable-certext
```

Comment:
I wonder if we should create an `--enable-wolfclu` option on the wolfSSL side. Not essential for this PR getting merged, just an idea.

Reply:
https://github.com/wolfSSL/wolfssl/pull/4362